providers/os/connection/container/auth/auth.go

```diff
@@ -4,36 +4,39 @@
 package auth
 
 import (
+	"strings"
+
+	"github.com/awslabs/amazon-ecr-credential-helper/ecr-login"
 	"github.com/google/go-containerregistry/pkg/authn"
 	"github.com/rs/zerolog/log"
-	"go.mondoo.com/cnquery/v10/logger"
-	"go.mondoo.com/cnquery/v10/providers-sdk/v1/vault"
-	"go.mondoo.com/cnquery/v10/providers/os/connection/container/image"
+	"go.mondoo.com/cnquery/v10/providers/os/connection/container/acr"
+)
+
+const (
+	acrIndicator = ".azurecr.io"
+	ecrIndicator = ".ecr."
```

Comment:
we should add a helper for gcp as well

Reply:
yep, +1

lets do it in a future PR to also understand how GCP registries work and if we need do build anything or there's a go package out there to help us (like the ECR one)